### PR TITLE
devel/qt5: Add cheribsd patch.

### PR DESCRIPTION
**Pull request by bukinr**
The patch is generated from [1].

[1] https://github.com/bukinr/qtbase/commit/47229b15c46c3f5f466f3f75c4cc25b6144f7233

### CODE REVIEW

File: devel/qt5-core/Makefile [hunk 1]
```diff
@@ -26,7 +33,11 @@ CONFIGURE_ARGS=	-no-accessibility -no-gif -no-libpng -no-libjpeg \
 		-no-egl -no-evdev \
 		-no-fontconfig -no-freetype -no-gtk -no-harfbuzz \
 		-no-libudev -no-xcb -no-xcb-xlib \
-		-no-xkbcommon -no-libinput
+		-no-xkbcommon -no-libinput -no-glib -no-mimetype-database
```
Comment (kwitaszczyk) [inline devel/qt5-core/Makefile:36]: We still need to revert this.

File: devel/qt5-core/Makefile [hunk 2]
```diff
@@ -12,7 +12,14 @@ LIB_DEPENDS=	libdouble-conversion.so:devel/double-conversion \
 		libpcre2-posix.so:devel/pcre2 \
 		libzstd.so:archivers/zstd
 
-USES=		compiler:c++11-lang gnome perl5 qmake:no_env qt-dist:5,base
+USES=		compiler:c++11-lang perl5 qmake:no_env qt-dist:5,base
+
+.include <bsd.port.options.mk>
```
Comment (kwitaszczyk) [inline devel/qt5-core/Makefile:17]: Is it needed?

File: devel/qt5-core/Makefile (hunk 1 above)
Comment (bukinr) [reply devel/qt5-core/Makefile]: Reverted

File: devel/qt5-core/Makefile (hunk 2 above)
Comment (bukinr) [reply devel/qt5-core/Makefile]: needed for ${ARCH}, but I moved these down after include bsd.port.mk

**Pull request merged**